app/templating/summary/question.py

```diff
@@ -6,14 +6,14 @@
 
 class Question:
 
-    def __init__(self, question_schema, answer_store, metadata):
+    def __init__(self, question_schema, answer_store, metadata, schema):
```

Comment:
What's the difference between question_schema and schema?

Reply:
`question_schema` is a 'fragment' (n.b. contentious term) of the whole `schema`, but there are places that the whole schema is needed to evaluate if something's in a repeating group

Reply:
thats what I thought, just wanted to be sure I got it